.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+### Description
+
+### What works
+
+### What was not tested
+
+### Test
+Operating system (and linux kernel version): 
+Hardware (GPU/CPU):
+
+### Ready for review
+[ ] Script tested as a regular phoenicis user and working (if you have a problem -> draft and ask for help).
+[ ] json-align and eslint run accorded to the [documentation](https://phoenicisorg.github.io/scripts/General/tools/). 
```

Comment:
Maybe use a code environment for `json-align` and `eslint run`

Reply:
`run` is not in the code environment

Reply:
Why should it be ? You run eslint and json-align.

Reply:
Ok additionally: `... according to...`